src/main/java/com/robothand/highqualitybot/Config.java

```diff
@@ -11,30 +14,27 @@
  * Reads in properties from a specified configuration file, and provides access to the values.
  */
 public class Config {
-    private final File configFile;
-
-    public String TOKEN;
+    String TOKEN;
     public String PREFIX;
+    public String OWNERID;
+    public ArrayList<Command> ALLOWED;
+    public ArrayList<Command> DISALLOWED;
+    public boolean ALLOWEDHASPRECEDENCE;
+    public ArrayList<String> PERMGROUPS;
 
 
     public Config(String path) throws FileNotFoundException {
```

Comment:
Yeah, this is more a relic from thinking about having separate configs for different guilds, I certainly agree with making those fields static. If we were to try to add in those other configs down the road, they'd likely be a separate class this one interacts with anyhow